src/application/t2d/generate_features_and_write_to_disk.py

```diff
@@ -108,10 +111,25 @@
     # Version table with current date and time
     # prefix with user name to avoid potential clashes
     current_user = Path().home().name + "_"
-    file_prefix = current_user + f"psycop_t2d_{time.strftime('%Y_%m_%d_%H_%M')}"
+    file_prefix = current_user + f"_{time.strftime('%Y_%m_%d_%H_%M')}"
+
+    # Create directory to store all files related to this run
+    sub_dir = SAVE_PATH / file_prefix
```

Comment:
```suggestion
    sub_dir = SAVE_PATH / f"{time.strftime('%Y_%m_%d_%H_%M')}"
```

Reply:
It was a bit hard for me to read with sub_dir using file_prefix in its name, but it not actually pointing to the file :-)